lib/internal/modules/esm/initialize_import_meta.js

```diff
@@ -71,6 +72,17 @@ function initializeImportMeta(meta, context, loader) {
 
   meta.url = url;
 
+  ObjectDefineProperty(meta, 'require', {
+    __proto__: null,
+    enumerable: true,
+    configurable: true,
```

Comment:
This probably should be writable as well, there might already be some code out there polyfilling this.